QMMHarmonyShimmer/QMMHarmonyShimmer.cs

```diff
@@ -33,9 +33,18 @@ public static class QMMHarmonyShimmer
         [Obsolete("Should not be used!", true)]
         public static void Initialize()
         {
-            ApplyHarmonyPatches();
-            InitAssemblyResolver();
-            ApplyShims();
+            try
+            {
+                ApplyHarmonyPatches();
+                InitAssemblyResolver();
+                ApplyShims();
```

Comment:
Do we have logs that mark when each of these methods begins?

Reply:
For `ApplyShims` yes, the others are very small, and the `Logger.LogFatal(ex.StackTrace)` should make it pretty clear what method the error is originating from.